.github/workflows/ci.yml

```diff
@@ -5,32 +5,40 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  schedule:
+    # Run every Sunday at midnight
+    - cron:  '0 0 * * 0'
+
+defaults:
+  run:
+    shell: bash -l {0}
 
 jobs:
   build:
     runs-on: ubuntu-18.04
     strategy:
+      fail-fast: false
       matrix:
         include:
           # Oldest supported versions
           - enable_cuda: true
-            cuda: 10.2.89
-            gcc: 8.5.0
-            nvcc: 10.2
-            python: 3.8
-            pytorch: 1.8.0
+            cuda: "10.2.89"
+            gcc: "8.5.0"
+            nvcc: "10.2"
+            python: "3.7"
+            pytorch: "1.11.0"
```

Comment:
Does this mean we're no longer trying to support any version before 1.11?

Reply:
Yes, only PyTorch 1.11. `conda-forge` has pinned the version of PyTorch (https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2812), so there will be no more packages for the older versions.